project/plugin-scalajs.sbt

```diff
@@ -1,2 +1,2 @@
-addSbtPlugin("org.scala-js"       % "sbt-scalajs"              % "1.5.1")
+addSbtPlugin("org.scala-js"       % "sbt-scalajs"              % "1.8.0")
```

Comment:
Necessary for Scala 2.13.8.  It's backward compatible, but will force all clients onto Scala.js >= 1.8.0.  That's the way most libraries are handling it.